### ISSUE TITLE
[BUG] Staff hub - white line between global nav and header image.

### ISSUE THREAD

**Issue by tblundell** [unimelb/design-system-issues#47]
In staff.unimelb, the class="headerless" has a 1px white border. This is noticeable when slotted between the nav and a page with a main header image. 
See https://staff.unimelb.edu.au/governance/projects/current-projects/information-technology/project-evolve#2020

## Expected behaviour

The global Nav and the header image would touch, without the white line.

## Current behaviour
<img width="1405" alt="Screen Shot 2019-12-10 at 12 13 16 pm" src="https://user-images.githubusercontent.com/943446/70486579-7dc5d800-1b46-11ea-9fde-f2fe5d6f787f.png">

## Steps to reproduce

1. load the page on Desktop
2. without scrolling, above the image is white line 

### Affected browsers

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] IE 11
- [X] Safari 8+
- [ ] iOS 8.4+
- [ ] Android 4.4+

**Comment by dkevey** [comment]
Hi @tblundell I can see that page has hardcoded elements on that page: 
![image](https://user-images.githubusercontent.com/1305804/70493714-c4bec800-1b5c-11ea-8344-4a99591d8be2.png)

and that the The staff hub templates have the `<div class="headerless"></div>` element is causing the issue. The Staff Hub templates aren't designed to have hardcoding like this and only the templates that are listed here are supported https://staff.unimelb.edu.au/web-support/staff-hub-support. Not sure this is something we would fix via CSS.
![image](https://user-images.githubusercontent.com/1305804/70493764-e4ee8700-1b5c-11ea-9837-55ca40a8c084.png)

**Comment by tblundell** [comment]
@dkevey ahhh, thanks. Im not the maintainer of this site but I do/did have edit access. I'll have a look (I noticed this when checking our link to the site). The line is not a showstopper anyway - more a consistency thing.

BTW: Using the inspector, I just tested changing a page with an image header (eg LMS website) to headless, and this displays the same line - not that you would really want that - so as you pointed out, headless would do this anywhere when not used correctly.

Thanks again.
